core/src/OpenMP/Kokkos_OpenMP_Exec.cpp

```diff
@@ -429,7 +429,9 @@ void OpenMP::print_configuration(std::ostream &s, const bool /*verbose*/) {
   const bool is_initialized = Impl::t_openmp_instance != nullptr;
 
   if (is_initialized) {
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_3
     Impl::OpenMPExec::verify_is_master("OpenMP::print_configuration");
+#endif
```

Comment:
I don't think this needs to stay. Essentially the verify_is_master is there to prevent you from doing this:
```c++
#pragma omp parallel
{
   Kokkos;:parallel_for(N, functor);
}
```

Or do I misread that?

Reply:
I think you are right, I have been overzealous with this one. This should probably become deprecated after other changes from #4119 land.